doc/sphinx/Pacemaker_Administration/pcs-crmsh.rst

```diff
@@ -0,0 +1,441 @@
+Quick Comparison of pcs and crm shell
+-------------------------------------
+
+``pcs`` and ``crm shell`` are two popular higher-level command-line interfaces
+to Pacemaker. Each has its own syntax; this chapter gives a quick comparion of
+how to accomplish the same tasks using either one. Some examples also show the
+equivalent command using low-level Pacmaker command-line tools.
+
+These examples show the simplest syntax; see the respective man pages for all
+possible options.
+
+Show Cluster Configuration and Status
+#####################################
+
+.. topic:: Show Configuration (Raw XML)
+
+   .. code-block:: none
+
+      crmsh     # crm configure show xml
+      pcs       # pcs cluster cib
+      pacemaker # cibadmin -Q
+
+.. topic:: Show Configuration (Human-friendly)
+
+   .. code-block:: none
+
+      crmsh # crm configure show
+      pcs   # pcs config
+    
+.. topic:: Show Cluster Status
+
+   .. code-block:: none
+
+      crmsh     # crm status
+      pcs       # pcs status
+      pacemaker # crm_mon -1
+
+Manage Nodes
+############
+
+.. topic:: Put node "pcmk-1" in standby mode
+
+   .. code-block:: none
+
+      crmsh     # crm node standby pcmk-1
+      pcs-0.9   # pcs cluster standby pcmk-1
+      pcs-0.10  # pcs node standby pcmk-1
+      pacemaker # crm_standby -N pcmk-1 -v on
+
+.. topic:: Remove node "pcmk-1" from standby mode
+
+   .. code-block:: none
+
+      crmsh     # crm node online pcmk-1
+      pcs-0.9   # pcs cluster unstandby pcmk-1
+      pcs-0.10  # pcs node unstandby pcmk-1
+      pacemaker # crm_standby -N pcmk-1 -v off
+
+Manage Cluster Properties
+#########################
+
+.. topic:: Set the "stonith-enabled" cluster property to "false"
+
+   .. code-block:: none
+
+      crmsh     # crm configure property stonith-enabled=false
+      pcs       # pcs property set stonith-enabled=false
+      pacemaker # crm_attribute -n stonith-enabled -v false
+
+Show Resource Agent Information
+###############################
+
+.. topic:: List Resource Agent (RA) Classes
+
+   .. code-block:: none
+
+      crmsh    # crm ra classes
+      pcs      # pcs resource standards
+      pacmaker # crm_resource --list-standards
+
+.. topic:: List Available Resource Agents (RAs) by Standard
+
+   .. code-block:: none
+
+      crmsh     # crm ra list ocf
+      pcs       # pcs resource agents ocf
+      pacemaker # crm_resource --list-agents ocf
+
+.. topic:: List Available Resource Agents (RAs) by OCF Provider
+
+   .. code-block:: none
+
+      crmsh     # crm ra list ocf pacemaker
+      pcs       # pcs resource agents ocf:pacemaker
+      pacemaker # crm_resource --list-agents ocf:pacemaker
+
+.. topic:: List Available Resource Agent Parameters
+
+   .. code-block:: none
+
+      crmsh     # crm ra meta IPaddr2
```

Comment:
Better use `crm ra info`, or there will be warning `WARNING: This command 'meta' is deprecated, please use 'info'`

Reply:
Thanks!